include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -202,21 +202,23 @@ class IExecutionProvider {
 
   /**
      Indicate whether the graph capturing mode (e.g., cuda graph) is enabled for
-     the provider. Currently only CUDA execution provider supports it.
+     the provider. Currently only CUDA/TensorRT/Rocm execution providers support it.
    */
   virtual bool IsGraphCaptureEnabled() const { return false; }
 
   /**
      Indicate whether the graph has been captured and instantiated. Currently
-     only CUDA execution provider supports it.
+     only CUDA/TensorRT/Rocm execution providers support it.
    */
   virtual bool IsGraphCaptured() const { return false; }
 
   /**
-     Run the instantiated graph. Currently only CUDA execution provider supports
-     it.
+     Run the instantiated graph. Currently only CUDA/TensorRT/Rocm execution providers
+     support it.
    */
-  virtual common::Status ReplayGraph() { return Status::OK(); }
+  virtual common::Status ReplayGraph(const onnxruntime::RunOptions& /*run_options*/) {
```

Comment:
Could only pass an integer of graph annotation ID instead of the whole run options.
Unless we have plan to support other options.

Reply:
Cheng suggests passing RunOptions but currently we only use annotation id.